websocketOutportDriver/utils_test.go

```diff
@@ -0,0 +1,46 @@
+package websocketOutportDriver_test
```

Comment:
This file literally doesn't test any productive code.
You are testing some "tricks" you also do in `webSocketSender`, using built-in functions(map slice, delete, etc.), not your defined functions. I don't see a strong reason why we should keep this file

Reply:
removed